ex0.py

```diff
@@ -0,0 +1,19 @@
+from PIL import Image
+
+
+def decipher(image_path):
+    #  decipher the hidden message inside the image
```

Comment:
Thats not proper documetation, use pycharm format, sphinx, google, or whatever you would like, but be consistent